cmd/downloader/main.go

```diff
@@ -245,12 +246,26 @@ func Downloader(ctx context.Context, logger log.Logger) error {
 			return err
 		}
 	}
-	cfg, err := downloadercfg.New(ctx, dirs, version, torrentLogLevel, downloadRate, uploadRate, torrentPort, torrentConnsPerFile, torrentDownloadSlots, staticPeers, webseedsList, chain, true, dbWritemap)
+	cfg, err := downloadercfg.New(
+		ctx,
+		dirs,
+		version,
+		torrentLogLevel,
+		downloadRate,
+		uploadRate,
+		torrentPort,
+		torrentConnsPerFile,
+		torrentDownloadSlots,
+		staticPeers,
+		webseedsList,
+		chain,
+		dbWritemap,
+	)
 	if err != nil {
 		return err
 	}
 
-	cfg.ClientConfig.PieceHashersPerTorrent = dbg.EnvInt("DL_HASHERS", 32)
+	cfg.ClientConfig.PieceHashersPerTorrent = dbg.EnvInt("DL_HASHERS", runtime.NumCPU())
```

Comment:
In my head hashers are not cpu-bouned: they are mmap-read-page-faults-bounded. 
This is reason why it's not linked to NumCPU.
On server with much cpus and slow disks (cloud drives) - it can lead to `golang's runtime panic - 10K threads exeeded` (because this limit is `per-file` and we have many files). 
Actually we need global limit for hashers amount - but let's do it in another (low-prio future) tasks.

Reply:
Thanks. I was curious as to the value and knew there'd a good reason.